portfolio/src/main/webapp/js/scripts.js

```diff
@@ -206,4 +207,28 @@ return [
     elementType: 'labels.text.stroke',
     stylers: [{color: '#17263c'}]
   }];
+}
+
+//geo chart API
+google.charts.load('current', {
+  'packages':['geochart'],
+  'mapsApiKey': 'AIzaSyBd6L5aVm5z-MWG4eBEiG1V4WF0pBn7LiM'
```

Comment:
We probably want to avoid posting API keys, since anyone can see scrape/them. That said, the solutions to this may be a bit more cumbersome than they're worth for this exercise.. Let's chat about this in one of our syncs.